components/logins/android/metrics.yaml

```diff
@@ -2,31 +2,36 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# This file defines the metrics that will be gathered for the "logins" storage component.
-# These are emitted for all users of the component.  Additional metrics specific to the
-# *syncing* of logins are defined in a separate "sync_ping" package.
+# This file defines the metrics that will be gathered for the "logins"
+# storage component.
+# These are emitted for all users of the component.  Additional metrics
+# specific to the *syncing* of logins are defined in a separate "sync_ping"
+# package.
```

Comment:
Just a comment for visibility for the a-s team: These whitespace-only changes here are due to the fact that `glean_parser` now runs `yamllint` over the input.